sample_apps/README.md

```diff
@@ -42,3 +42,45 @@ that demonstrate how to implement the Certifier APIs on different platforms.
 - **analytics_example/** - This is an example of Privacy Preserving data analytics with
   the Certifier framework. This sample application demonstrates how to conduct
   privacy preserving data analytics with the Certifier Framework on Open Enclave (OE).
+
+# run_example.sh - Setup and run test for sample apps program
+
+This document is a user's guide to the run_example.sh script that you can use
```

Comment:
Pulled-up `simple_app/README.md` into this `README.md`. 

The User's guide instructions are common from now on to all sample programs that will be brought under this script.

Reply:
You can see the consolidated README.md [here](https://github.com/vmware-research/certifier-framework-for-confidential-computing/tree/agurajada/64-Common-run_example.sh/sample_apps).